MemoryMgr.h

```diff
@@ -6,17 +6,28 @@
 
 #define WRAPPER __declspec(naked)
 #define DEPRECATED __declspec(deprecated)
+#define WRAPARG(a) ((int)a)
+
+#ifdef _MSVC_VER
```

Comment:
It's `_MSC_VER`.

Reply:
That was a fairly easy definition to confuse (anyway I changed that)